api/src/wfl/module/covid.clj

```diff
@@ -263,6 +237,35 @@
                     (->> (jdbc/insert! tx terra-executor-table)))]
     [terra-executor-type (-> items first :id)]))
 
+(defn import-snapshot!
+  "Return snapshot reference for SNAPSHOT_ID imported to WORKSPACE."
+  [{:keys [workspace] :as _executor}
+   {:keys [snapshot_id] :as _snapshot}]
+  (rawls/create-snapshot-reference workspace snapshot_id))
```

Comment:
Note we need the name to update the method config

Reply:
I'll see what Rex did in his open PR with appending the timestamp instead of a random suffix.  I'd be happier with that middle ground if it's feasible.

Reply:
Also, because this returns the whole snapshot reference, downstream functions will have access to the snapshot reference name.

Reply:
Great.

Reply:
Closing the loop, I did end up using the name that the snapshot was created with as you originally requested.  When Rex's changes are merged, that name will have the formatted timestamp as its suffix, so a double timestamp suffix would be overkill.